src/components/BulkEnrollmentModal/index.jsx

```diff
@@ -0,0 +1,219 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field, reduxForm, SubmissionError } from 'redux-form';
+import {
+  Alert, Button, Icon, Modal,
+} from '@edx/paragon';
+import TextAreaAutoSize from '../TextAreaAutoSize';
+import CsvUpload from '../CsvUpload';
+
+class BulkEnrollmentModal extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.errorMessageRef = React.createRef();
+    this.modalRef = React.createRef();
+
+    this.handleModalSubmit = this.handleModalSubmit.bind(this);
+
+    this.state = { open: true };
```

Comment:
I think current best practice is to put this below the constructor as
`state = { open: true };`
But I also think Stank and I disagree about that, so, take it or leave it.